vernac/vernacentries.ml

```diff
@@ -1420,13 +1420,13 @@ let vernac_coercion ~atts ref qidst =
   let ref' = smart_global ref in
   match qidst with
   | Some (qids, qidt) ->
-     let (local, poly), reversible =
-       Attributes.parse Notations.(locality ++ polymorphic ++ reversible) atts in
```

Comment:
For easier transition, you should add a warning rather than not parsing the polymorphic attribute. In a later version we will actually fail.

Reply:
No, unsupported attribute is already a warning.
Also here the attribute was a noop AFAICT so removing it is backwards compatible.

Reply:
It's a warning that errors by default, but OK.